webapp/tests/forms/flows/test_lotse_step_chooser.py

```diff
@@ -3,6 +3,7 @@
 
 from werkzeug.datastructures import ImmutableMultiDict
 
+from app.utils import VERANLAGUNGSJAHR
```

Comment:
Why did you decide for a TEST_VERANLAGUNGSJAHR in Erica but not in webapp?

Reply:
Mh, good question. Didn't think about it too much. Which method do you prefer?

Reply:
Let's use the same constant for both tests and code